soft4pes/model/conv/inverter.py

```diff
@@ -0,0 +1,100 @@
+""" 
+n-level inverter model.
```

Comment:
First, let's name this "converter" instead of "inverter"

Reply:
All mentions of _inverter_ have been changed to _converter_